tests/serverpod_test_server/lib/src/endpoints/exception_approach.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/user_not_found_exception.dart';
+
+class ExceptionApproach extends Endpoint {
+  Future<String> checkUserExist(Session session, int userId) async {
```

Comment:
Here, it would be better to split this test up into multiple methods, to make it more clear what is going on.

E.g. `throwSerializableException`, `throwNormalException`, `throwExceptionWithMessage`, etc.